mac/scripts/preinstall

```diff
@@ -88,13 +91,14 @@ is_tesseract_installed () {
   return
 }
 
-is_jre11_installed () {
-  # The directory containing JRE11 was not found
-  if ! [ -d /Library/Java/JavaVirtualMachines/adoptopenjdk-11.jre ]; then
+is_jre_installed () {
+  # This command returns JRE directory for the correct version if it exists
+  /usr/libexec/java_home -F -v $jre_version
```

Comment:
More about this command [here](https://medium.com/notes-for-geeks/java-home-and-java-home-on-macos-f246cab643bd)